src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-capabilities.mdx

```diff
@@ -0,0 +1,275 @@
+
+To learn what capabilities a role has, go to the user management UI and view a specific role. The location of this UI depends on what [user model](/docs/accounts/original-accounts-billing/original-users-roles/overview-user-models) a user is on: 
+
+* Newer user model: from [one.newrelic.com](https://one.newrelic.com), click **[account dropdown](/docs/new-relic-solutions/get-started/glossary/#account-dropdown) &gt; Administration &gt; Access management &gt; Roles**.  
+* Original user model: from [one.newrelic.com](https://one.newrelic.com), click **[account dropdown](/docs/new-relic-solutions/get-started/glossary/#account-dropdown) &gt; Account settings &gt; Users and roles**.  
+
+## How capabilities relate to roles [#capabilities-roles]
+
+Groupings of capabilities are aggregated into roles. We have pre-built roles for common use cases, and you can build your own custom roles. 
+
+Our pre-built roles work differently depending on which of our [user models](/docs/accounts/original-accounts-billing/original-users-roles/overview-user-models) you're on: 
+
+* Our newer user model: [learn about roles](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts). 
+* Our original user model: [learn about roles](/docs/accounts/original-accounts-billing/original-users-roles/users-roles-original-user-model). 
+
+## Some important tips for using capabilities [#important-points]
+
+Some important points for understanding and using capabilities: 
+
+* **Capabilities can change.** The capabilities we expose in the UI are subject to change and reorganization. The capabilities in this doc were last updated June, 2022.     
+* **Some capabilities have overlapping functionality.** That is why selecting some capability checkboxes will automatically check or uncheck other boxes.
+* **Not all capabilities are exposed.** Some roles have hidden capabilities that are not visible in the UI and not available for addition to a custom role. For example, the **Organization manager** role has capabilities related to managing and configuring organization-level settings, but those are not available for addition to a custom role. 
+* **Capabilities don't affect querying.** Most capabilities apply to New Relic UI experiences and not to querying data. For example, if your capabilities restrict you from accessing the APM UI, you can still query APM-related data using the query builder if you have access to that account. If you require more firm data boundaries for some projects or users, one way to do this is by [adding accounts](/docs/accounts/accounts-billing/account-structure/add-accounts).
+
+## Capability definitions [#capability-definitions]
+
+Here's a composite screenshot of the capabilities we expose in the capabilities UI for the purpose of adding to custom roles. 
```

Comment:
I like the screenshot, but this chunk of text could probably be more interactive and answer more questions. Maybe instead of "Here's a composite screenshot" something like:

"This screenshot shows all of the possible capabilities that are viewable by all users. These capabilities can be applied to custom roles in any combination your org needs."

This could also be a good spot to explain which capabilities are **not** exposed here.

Reply:
Emphasis here on calling out "all possible capabilities" and "all users." Just so it's more obvious for the audience who and what this pertains to, even if a little repetitive.